Makefile

```diff
@@ -134,6 +139,13 @@ build/linux-mips-softfloat/%: LDFLAGS += -s -w
 build/linux-amd64-boringcrypto/%: GOENV += GOEXPERIMENT=boringcrypto CGO_ENABLED=1
 build/linux-arm64-boringcrypto/%: GOENV += GOEXPERIMENT=boringcrypto CGO_ENABLED=1
 
+# pkcs11
+build/linux-amd64-pkcs11/%: BUILD_ARGS += -tags pkcs11
+build/linux-amd64-pkcs11/%: CGO_ENABLED=1
+build/linux-arm64-pkcs11/%: BUILD_ARGS += -tags pkcs11
+build/linux-arm64-pkcs11/%: CGO_ENABLED=1
+build/linux-arm64-pkcs11/%: GOENV += CC=aarch64-linux-gnu-gcc
```

Comment:
The current test-linux runner doesn't have an aarch64 compiler, which is making runs fail :/

Leaving this comment to track, and in the meantime, removing it from the `all` target